bdsx/bds/packets.ts

```diff
@@ -44,17 +44,20 @@ export class DisconnectPacket extends Packet {
 
 @nativeClass(null)
 export class ResourcePacksInfoPacket extends Packet {
-    // unknown
+    @nativeField(bool_t)
+    forcedtoacceot:bool_t;
 }
 
 @nativeClass(null)
 export class ResourcePacksStackPacket extends Packet {
```

Comment:
@karikera This should be ResourcePackStackPacket instead of ResourcePacksStackPacket

Reply:
I just realised that my typing skills are beyond annoying

Reply:
Thats not your fault I think, it existed for a long time